### ISSUE TITLE
Missing Vue.navigateTo method promised by Typescript bindings

### ISSUE THREAD

**Issue by IlyaSemenov** [nativescript-vue/nativescript-vue#675]
### Version
2.6.4

### Reproduction link
[https://play.nativescript.org/?template=play-vue&id=Sc4pJM](https://play.nativescript.org/?template=play-vue&id=Sc4pJM)
### Platform and OS info
Android 7.0 / NativeScript 6.7.8 / macOS 10.15.5

### Steps to reproduce
```ts
import Vue from "nativescript-vue"

...

Vue.navigateTo(MyComponent)
```

### What is expected?
Navigation succeeds.

### What is actually happening?
TypeError: _nativescriptVue.default.navigateTo is not a function

---
The existence of global `Vue.navigateTo` method is falsely advertised by `index.d.ts`:

```ts
export interface NativeScriptVueConstructor extends VueConstructor<NativeScriptVue> {
    navigateTo: navigateTo
    navigateBack: () => void
```

Although this method is actually missing, it is **highly desired**, because in Vue Composition API (the new Vue 3 API which is also backported to Vue 2) there is no 'this' context (hence, there is no way to call `this.$navigateTo`).

**Comment by rigor789** [comment]
With the composition api you should be able to access `$navigateTo` from the prototype directly: `Vue.prototype.$navigateTo`. We had `Vue.navigateTo` at some point, and it has been removed with a bigger refactor that no longer allowed navigating without a `this` context - this is no longer present so adding back `Vue.navigateTo` should be simple. 

https://github.com/nativescript-vue/nativescript-vue/blob/1e6f9dc21fe5c4effeb5ce2a6b809cff03b34a96/platform/nativescript/plugins/navigator-plugin.js#L66

Would need to be changed to
```js
Vue.navigateTo = Vue.prototype.$navigateTo = function (component, options) {
```

(and same for `$navigateBack`).

PR is welcome!

**Comment by rigor789** [comment]
Released in v2.7.0
